Makefile

```diff
@@ -13,17 +13,16 @@ test: image
 	  codeclimate/codeclimate exec rake spec:all spec:benchmark
 
 citest:
-	docker run \
-	  --env CIRCLECI=$(CIRCLECI) \
-	  --env CIRCLE_BUILD_NUM=$(CIRCLE_BUILD_NUM) \
-	  --env CIRCLE_BRANCH=$(CIRCLE_BRANCH) \
-	  --env CIRCLE_SHA1=$(CIRCLE_SHA1) \
-	  --env CODECLIMATE_REPO_TOKEN=$(CODECLIMATE_REPO_TOKEN) \
-	  --entrypoint bundle \
-	  --volume $(PWD)/.git:/usr/src/app/.git:ro \
-	  --volume /var/run/docker.sock:/var/run/docker.sock \
-	  --volume $(CIRCLE_TEST_REPORTS):/usr/src/app/spec/reports \
-	  codeclimate/codeclimate exec rake spec:all spec:benchmark
+	docker run --entrypoint sh \
+	--env CIRCLECI=$(CIRCLECI) \
+	--env CIRCLE_BUILD_NUM=$(CIRCLE_BUILD_NUM) \
+	--env CIRCLE_BRANCH=$(CIRCLE_BRANCH) \
+	--env CIRCLE_SHA1=$(CIRCLE_SHA1) \
+	--env CODECLIMATE_REPO_TOKEN=$(CODECLIMATE_REPO_TOKEN) \
+	--volume $(PWD)/.git:/usr/src/app/.git:ro \
+	--volume /var/run/docker.sock:/var/run/docker.sock \
+	--volume $(CIRCLE_TEST_REPORTS):/usr/src/app/spec/reports \
+	codeclimate/codeclimate -c "bundle exec rake spec:all spec:benchmark && bundle exec codeclimate-test-reporter"
```

Comment:
💄 the continuation lines should be indented two spaces below the `docker run`, like they were.